runelite-client/src/main/java/net/runelite/client/plugins/reportbutton/ReportButtonPlugin.java

```diff
@@ -45,22 +45,22 @@
 import net.runelite.client.callback.ClientThread;
 import net.runelite.client.config.ConfigManager;
 import net.runelite.client.eventbus.Subscribe;
+import net.runelite.client.events.ConfigChanged;
 import net.runelite.client.plugins.Plugin;
 import net.runelite.client.plugins.PluginDescriptor;
 import net.runelite.client.task.Schedule;
 
 @PluginDescriptor(
 	name = "Report Button",
 	description = "Replace the text on the Report button with the current time",
-	tags = {"time", "utc"}
+	tags = {"time", "utc", "clock"}
 )
 public class ReportButtonPlugin extends Plugin
 {
 	private static final ZoneId UTC = ZoneId.of("UTC");
 	private static final ZoneId JAGEX = ZoneId.of("Europe/London");
-
-	private static final DateTimeFormatter DATE_TIME_FORMAT = DateTimeFormatter.ofLocalizedTime(FormatStyle.MEDIUM);
 	private static final DateFormat DATE_FORMAT = new SimpleDateFormat("MMM. dd, yyyy");
+	private static DateTimeFormatter format;
```

Comment:
This looks like it is not set if `ConfigChanged` is not fired? Also, why is this `static`?

Reply:
This is static due to the methods using it being static, I have no idea why the methods were made static at one point.

Reply:
Because they didn't depend on the state of a variable.